plasmapy/examples/langmuir_analysis.py

```diff
@@ -0,0 +1,89 @@
+# coding: utf-8
```

Comment:
If we call this file `plot_langmuir_analysis` (per current convention for our files), it's going to show the actual plots! See the example via Giles under tests. The state for the most recent commit as of now is [here](https://1202-46810954-gh.circle-artifacts.com/0/home/circleci/project/docs/_build/html/auto_examples/plot_dispersion_function.html#sphx-glr-auto-examples-plot-dispersion-function-py).